docs/first-step/run-executable.rst

```diff
@@ -4,9 +4,9 @@
 Build and run executable
 ------------------------
 
-In this section we will build and run ``foo`` executable. In case if IDE was
-used you can do it by opening project in IDE or using command line (it doesn't
-matter how project was generated before: using GUI or CLI version of CMake).
+In this section we will build and run the ``foo`` executable. You can do
+it by opening the project in an IDE or by using the command line (It doesn't
```

Comment:
> (It doesn't

Should it be lowercase `(it doesn't` ?

Reply:
hmm yeah probably.  ooh theres another "the" to add here, Ill update it.
Perhaps even you want to consider removing the parentheses alltogether?

Reply:
> Perhaps even you want to consider removing the parentheses alltogether?

Up to you, I'm okay with either.